aim/web/ui/src/components/ImagesSet/ImagesSet.d.ts

```diff
@@ -1,3 +1,5 @@
+import { IFocusedState } from '../../types/services/models/metrics/metricsAppModel';
```

Comment:
Please change it to import { IFocusedState } from 'types/services/models/metrics/metricsAppModel';

Reply:
already changed thanks